scripts/lib/CIME/case_st_archive.py

```diff
@@ -410,6 +484,39 @@ def restore_from_archive(case, rest_dir=None):
         shutil.copy(item, rundir)
 
 ###############################################################################
+def archive_last_restarts(case, archive_restdir, link_to_restart_files=False):
```

Comment:
Most of the refactor supports sharing code between the real short-term archiving process and the erp test (and potentially future tests). This function is the one piece of new code that is NOT part of the standard short-term archiving process.